solr/core/src/java/org/apache/solr/update/UpdateShardHandlerConfig.java

```diff
@@ -32,6 +32,15 @@ public class UpdateShardHandlerConfig {
           DEFAULT_METRICNAMESTRATEGY,
           DEFAULT_MAXRECOVERYTHREADS);
 
+  public static final UpdateShardHandlerConfig TEST_DEFAULT =
```

Comment:
Shouldn't a TEST_DEFAULT be declared in some sort of testing class or in the Test Framework jar ???

Reply:
That's debatable... because there is no obvious place where it would live exactly.  I suppose TestHarness *was* okay maybe but now that TestHarness has a competitor, ehh....

As it's just a "config" thing and not some heavy object, its home seems arbitrary to me -- not a problem that it's created in production.  I don't have a strong opinion though.